mms/context.py

```diff
@@ -77,3 +80,6 @@ def report_status(self, code, reason_phrase=None):
 
     def add_response_property(self, key, value):
```

Comment:
response header should be per request, we need update this API to support per request header

Reply:
True, we need that.